store/src/java/com/zimbra/cs/service/UserServlet.java

```diff
@@ -279,7 +279,7 @@ private void sendError(UserServletContext ctxt, HttpServletRequest req, HttpServ
             && (req.getMethod().equalsIgnoreCase("POST") || req.getMethod().equalsIgnoreCase("PUT"))) {
             resp.sendError(HttpServletResponse.SC_UNAUTHORIZED, L10nUtil.getMessage(MsgKey.errMustAuthenticate, req));
         } else {
-            if (ctxt.getAuthAccount() instanceof GuestAccount && ctxt.basicAuthAllowed() ) {
+            if (ctxt != null && ctxt.getAuthAccount() instanceof GuestAccount && ctxt.basicAuthAllowed() ) {
```

Comment:
Think it would be cleaner codewise to handle ctxt==null before this if branch (or in the if branch)